### DISCUSSION TITLE
My application keeps getting rejected

### DISCUSSION THREAD

**Discussion by PeterKoszorus** [community/community#143671] [GitHub Education]
### Hi everyone,

Hi, everyone I m trying to re-apply for the GitHub Education programme, but my application keeps getting rejected I have uploaded a screenshot of my proof of study in english where everything is displayed. I dont want to share the proof of study here as it contains sensitive data so if smbdy from GitHub will help me I will provide this proof to their hands.

**Comment by InWamos** [comment]
Share us the reason for rejection. We need more info on your situation(Without your personal data)

UPD: From the PRO badge I see on your profile, I assume you already have Copilot
